dags/mantaray/run_mantaray_jobs.py

```diff
@@ -31,17 +31,21 @@
   )
   xlml_jobs = yaml.safe_load(xlml_jobs_yaml)
 
-  # Create a DAG for PyTorch/XLA tests
-  pattern = r"^(ptxla|pytorchxla).*"
+  # Create two DAG for PyTorch/XLA tests
+  pattern = r"^(ptxla|pytorchxla_part1).*"
+  pattern2 = r"^(pytorchxla_part2).*"
```

Comment:
Should this be called "pattern_sd" and the patterns "pytorchxla_llm", "pytorchxla_sd" respectively? It looks like you intended to separate transformer vs diffuser runs.

Reply:
Ditto for workload_file_name_list and workload_file_name_list_2

Reply:
Thanks, @tengyifei, we want to split test to the following two part:
- `llama3-8b:v6e-8`, `llama3-70b:v5p-64`, `mixtral8_7b:v5p-32`;
- `llama3-70b:v5p-32`, `llama3-70b:v5p-16`, `sd2`;

I updated the second DAG name to a new one, which is not bond to sd2;

related internal chage has requested your review too

Reply:
ack. i suggest finding a less hacky way to do this. for example, maybe [1] helps?

[1]: https://stackoverflow.com/questions/56370720/how-to-control-the-parallelism-or-concurrency-of-an-airflow-installation

Reply:
thanks, cool, smart, your suggestion works in local test, updated based on it